gitlab2sentry/utils/gitlab_provider.py

```diff
@@ -221,7 +221,7 @@ def _get_default_mentions(self, project: Project) -> str:
         return ", ".join(
             [
                 f"@{member.username}"
-                for member in project.members.list()
+                for member in project.members_all.list(get_all=True)
```

Comment:
```suggestion
                for member in project.members_all.list(iterator=True)
```

Reply:
the `get_all` from the doc is out-dated

Reply:
Params for `.list()` are:
- `all` – If True, return all the items, without pagination
- `per_page` – Number of items to retrieve per request
- `page`– ID of the page to return (starts with page 1)
- `iterator` – If set to True and no pagination option is defined, return a generator instead of a list